src/app/components/demo-components.service.ts

```diff
@@ -52,7 +52,7 @@ export class SkyDemoComponentsService {
         icon: 'square-o',
         // tslint:disable-next-line
         summary:
-          `The action button component creates a large button with an icon, heading, and details.`,
```

Comment:
I'm not 100% sure what caused that. I don't know if I did a format on that file accidentally or what. I can see if I can fix that.